astlib/migrate_501_502.ml

```diff
@@ -0,0 +1,1262 @@
+module From = Ast_501
```

Comment:
`Option.map` is not available on OCaml `4.04`:

```suggestion
open Stdlib0
module From = Ast_501
```

Reply:
This should be applied to quite a few other `Migrate` modules then, as I created those two files by copying `astlib/migrate_501_500.ml` and hand-editing the only interesting cases.

Reply:
Yes, I've opened #417 with those modifications (sorry, I should have mentioned it here!)